lib/helpbase.gi

```diff
@@ -493,12 +493,16 @@ InstallValue(HELP_KNOWN_BOOKS, [[],[]]);
 
 
 InstallGlobalFunction(HELP_ADD_BOOK, function( short, long, dir )
-  local sortfun, str, hnb, pos;
+  local main, sortfun, str, hnb, pos;
+  main := ["tutorial", "reference", "changes"];
+  if ForAny( GAPInfo.RootPaths,
+             rp -> IsDirectoryPath( Concatenation( rp, "/doc/dev" ) ) ) then
+    Add( main, "development" );
+  fi;
```

Comment:
Why is this needed? can't you just add `"development"` to the list inside `sortfun` and be done with it?

Reply:
This is needed because we do not include this manual in the GAP distribution. Done outside `sortfun` accordingly to your comment in https://github.com/gap-system/gap/pull/1194